### PR TITLE
Publish binary via drone

### PR DESCRIPTION
**Pull request by nikhilpandit**
- Modified Makefile to allow making releases
- Modified drone.yml to push the latest release on master branch to Github

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,29 +1,57 @@
...
-	go test $@ -test.v
+	@echo "TESTING..."
+	@go test $@ -test.v
 endif
-	go install -v $@
 
-install:
-	sudo cp $(GOPATH)/bin/batchtail /usr/local/bin/batchtail
+build/*: version.go
+version.go: VERSION
```
Comment (azylman) [inline Makefile]: In the past when we've depended on an auto-generated file, we put the file under `.gitignore`

**Comment by azylman** [comment]
lgtm

File: Makefile (hunk 1 above)
Comment (nikhilpandit) [inline Makefile]: ah, this file was auto-generated. I just wasn't sure if it goes in .gitignore. I'll add it in.

**Comment by nikhilpandit** [comment]
@azylman - removed `version.go` and added it to `.gitignore`

**Comment by azylman** [comment]
lgtm

**Pull request merged**